lib/glific/wa_groups.ex

```diff
@@ -0,0 +1,114 @@
+defmodule Glific.WAGroups do
```

Comment:
There is `whatsapp_group.ex` and `wa_groups.ex` (for wa_managed_phone functions). Do we need a better naming?